Runtime/CesiumCameraController.cs

```diff
@@ -0,0 +1,1145 @@
+
+            this.FlyToLocationEarthCenteredEarthFixed(
+                destinationECEF,
+                yawAtDestination,
+                pitchAtDestination,
+                canInterruptByMoving);
+        }
+
+        /// <summary>
+        /// Begin a smooth camera flight to the given  WGS84 longitude in degrees (x),
+        /// latitude in degrees (y), and height in meters (z) such that the camera ends 
+        /// at the specified yaw and pitch.
+        /// </summary>
+        /// <remarks>
+        /// The characteristics of the flight can be configured with 
+        /// <see cref="CesiumCameraController.flyToAltitudeProfileCurve"/>,
+        /// <see cref="CesiumCameraController.flyToProgressCurve"/>, 
+        /// <see cref="CesiumCameraController.flyToMaximumAltitudeCurve"/>,
+        /// <see cref="CesiumCameraController.flyToDuration"/>, and
+        /// <see cref="CesiumCameraController.flyToGranularityDegrees"/>.
+        /// </remarks>
+        /// <param name="destination">The longitude (x), latitude (y), and height (z) of the destination.</param>
+        /// <param name="yawAtDestination">The yaw of the camera at the destination.</param>
+        /// <param name="pitchAtDestination">The pitch of the camera at the destination.</param>
+        /// <param name="canInterruptByMoving">Whether the camera flight can be interrupted with movement inputs.</param>
+        public void FlyToLocationLongitudeLatitudeHeight(
+            Vector3 destination,
+            float yawAtDestination,
+            float pitchAtDestination,
+            bool canInterruptByMoving)
+        {
+            double3 destinationCoordinates = new double3()
+            {
+                x = destination.x,
+                y = destination.y,
+                z = destination.z
+            };
+            double3 destinationECEF =
+                CesiumTransforms.LongitudeLatitudeHeightToEarthCenteredEarthFixed(
+                    destinationCoordinates);
+
+            this.FlyToLocationEarthCenteredEarthFixed(
+                destinationECEF,
+                yawAtDestination,
+                pitchAtDestination,
+                canInterruptByMoving);
+        }
+
+        #endregion
+
+        #region Dynamic clipping plane adjustment
+
+        private void UpdateClippingPlanes()
+        {
+            if (this._camera == null)
+            {
+                return;
+            }
+
+            // Raycast from the camera to the Earth's center and compute the distance.
+            float height = 0.0f;
+            if (!this.RaycastTowardsEarthCenter(out height))
+            {
+                return;
+            }
+
+            float nearClipPlane = this._initialNearClipPlane;
+            float farClipPlane = this._initialFarClipPlane;
+
+            if (height >= this._dynamicClippingPlanesMinHeight)
+            {
+                farClipPlane = height + (float)(2.0 * CesiumEllipsoid.GetMaximumRadius());
```

Comment:
Is a 2x max radius necessary? I think anything beyond one radii would be on the back side of the globe anyway, right? Of course if you have stuff you want to see in orbit, then either one might be insufficient, but there's probably not a whole lot we can do about that given Unity's severe clipping plane limitations.

Reply:
Yeah I thought that one radii would be enough, but when I did it with just one radii, the edge of the Earth was noticably "bouncing" as it zoomed out.
![OnlyRadius](https://user-images.githubusercontent.com/32226860/211608482-3179bf5e-5df7-4076-967e-6ee569a2b473.gif)

So I multiplied it by 2 so that the bouncing wouldn't happen at that a distance where the Earth was still relatively sizeable on the screen.

![TwiceRadius](https://user-images.githubusercontent.com/32226860/211608775-0f652438-ecd3-42cf-8778-e3039eb8e818.gif)

Of course, once it gets extremely small we run into the same effect, but it takes longer to get there.
 
![TwiceRadiusVeryFarAway](https://user-images.githubusercontent.com/32226860/211609296-5dd53aac-b012-4249-bd0f-a819c2b97841.gif)

If there's a more robust way of calculating this scalar, or if it's better to leave it as one radii and just communicate how to set up rendering environments for objects that are far away, let me know.

Reply:
Ok, I played around with this and... I have no idea what Unity is doing. But sticking to 2x for now is fine.